package.json

```diff
@@ -12,19 +12,23 @@
     "lint-staged": "^12.4.3",
     "prettier": "^2.8.8"
   },
-  "packageManager": "yarn@4.0.1",
+  "dependencies": {
+    "@changesets/cli": "^2.26.2"
```

Comment:
It's a dev dep right?

Reply:
https://www.npmjs.com/package/@changesets/cli docs for whatever reason say to add as a normal dep in a monorepo